.github/workflows/regression-tests.yml

```diff
@@ -168,6 +168,15 @@ jobs:
             cc: clang
             extra_name: "clang | "
 
+          - sim: icarus             # with gcov/gcovr
+            sim-version: v10_3
+            lang: verilog
+            python-version: 3.8
+            os: ubuntu-latest
+            extra_name: "gcovr | "
+            cxx: "ccache g++"
+            cc: "ccache gcc"
```

Comment:
Upon evaluation I think it's doing something like `CC=ccache gcc` which only binds `ccache` to `CC`. You might need to use escaped quotes.

https://github.com/cocotb/cocotb/pull/2155/checks?check_run_id=1299131982#step:23:53